active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,335 @@
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedPipStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty DefaultPipStyleProperty{ get; };
+}
+```
+
+![](images/pager-control-vertical-pips-scroll-ex.jpg)
+
+
+# Input & Accessibility
+
+## Keyboard
+
+- Tab navigation into the PipsPager proceeds through the first actionable item. This would be the previous button (if not Collapsed) or the first pip. Any directional buttons can be navigated to via tab if they are not Collapsed and will be visible on focus.
+- The arrow keys can be used to navigate between the pips and directional buttons. Independent of the orientation set, the left and up arrow keys will change focus to the previous pip or to the previous button and the right and down arrow keys will change focus to the next pip or the next button.
+
+## Gamepad
+
+- The PipsPager can be navigated to via spatial navigation. The selected pip will automatically take focus and the pips will be selected when navigated to and focused. Independent of the PipsPagerButtonVisibility settings, the navigation buttons will not be visible to the user. 
+  - In horizontal orientation left and right spatial navigation will focus and select the pips. The PipsPager will be navigated out of when the user navigates up or down or when the user attempts to navigate past the first or last page. We recommend that developers do not place UI directly to the left or right of a horizontally oriented PipsPager. 
+  - In vertical orientation up and down spatial navigation will focus and select the pips. The PipsPager will be navigated out of when the user navigates left or right or when the user attempts to navigate past the first or last page. We recommend that developers do not place UI directly above or below a vertically oriented PipsPager. 
+
+## Touch
+
+For touch-optimized experiences, we recommend that developers integrate the PipsPager with a view control (i.e. FlipView, see guidance [above](#integrate-pipsPager-with-a-collection-control)) to take advantage of on-content pagination with touch.
+
+The user can also touch to select the individual pip if able.
+
+## Screen readers
+
+- When the focus is on the control, the screen reader will announce "pager".
+- When the focus is on the previous or next button the screen reader will announce "first page", "last page", "next page", or "previous page". 
+  - If the buttons have text properties set by the developer, the screen reader will announce that text instead of the default announcement.
+- When the focus is on a pip, the screen reader will announce "page x of y".
```

Comment:
What if the number of pages is infinite?

Reply:
@oldnewthing @gabbybilka the current (incorrect) behaviour is that the Narrator will announce "
There is an [issue](https://github.com/microsoft/microsoft-ui-xaml/issues/4133) regarding this problem.

Reply:
I added this note in the latest update, thanks. I'll leave this thread open as an FYI to the open bug.